docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Set default values for required env vars
+export DB_HOST=${DB_HOST:-"postgres"}
+export DB_PORT=${DB_PORT:-"5432"}
+export DB_NAME=${DB_NAME:-"waltz"}
+export DB_USER=${DB_USER:-"waltz"}
+export DB_PASSWORD=${DB_PASSWORD:-"waltz"}
+export DB_SCHEME=${DB_SCHEME:-"waltz"}
+export WALTZ_FROM_EMAIL=${WALTZ_FROM_EMAIL:-"help@finos.org"}
+export WALTZ_BASE_URL=${WALTZ_BASE_URL:-"http://127.0.0.1:8080/"}
```

Comment:
@ljubon should this be equal to `http://127.0.0.1:8080/waltz-web`? If you remember, when war is deployed to tomcat, the name of the war becomes the root of the service. But if this var is used differently, than i guess it is fine.

Reply:
as you said, the war file name `waltz-web` will be added ...so in practice URL will have that addition in order to open dashboard

Reply:
@pavlovic-ivan this is resolved now, but please let's continue further discussions on new PR https://github.com/ljubon/waltz/pull/2

I've accidentally rebased on upstream instead of origin master branch -.-